xonsh/readline_shell.py

```diff
@@ -29,6 +29,7 @@
     ansi_color_style,
 )
 from xonsh.prompt.base import multiline_prompt
+from xonsh.completers.tools import RichCompletion
```

Comment:
I think this need to be `import xonsh.completers.tools` to get the automatic lazy imports

Reply:
What (/where) are lazy imports in xonsh?

Reply:
They are generated as part of amalgamate

Reply:
I can't find __amalgamate__.py anywhere, where is it supposed to be?

Reply:
Yeah, I think there are issues with `pip install` not generating it right now. It should be in the install location.

Reply:
But the fix here is to say `import xonsh.completers.tools` or `import xonsh.completers.tools as xct` and then use this name below

Reply:
done